app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden:
+                 bool = False) -> None:
```

Comment:
There is a syntax error in the `__init__` method signature. The `hidden` parameter should be declared as `hidden: bool = False` without splitting the type annotation and default value across two lines.